src/Farmer/Arm/Network.fs

```diff
@@ -621,6 +621,8 @@ type NetworkInterface =
 
 
 
+
```

Comment:
Blank lines here too?

Reply:
i'll delete the blanks and re-run fantomas and if they get added back it must just be for standard formatting reasons